.rubocop.yml

```diff
@@ -7,7 +7,7 @@ inherit_mode:
 # Defaults can be found here: https://github.com/bbatsov/rubocop/blob/master/config/default.yml
 
 AllCops:
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.2
```

Comment:
Changing this line causes many lint errors. I think it should be remain as it-is for now.